src/Uno.Templates/content/unoapp/MyExtensionsApp.1/GlobalUsings.cs

```diff
@@ -1,6 +1,9 @@
 ﻿//-:cnd:noEmit
 global using System.Collections.Immutable;
 global using Microsoft.Extensions.DependencyInjection;
+global using Uno.Extensions.Serialization;
```

Comment:
Uno.Extensions.Serialization can't always be included - for the blank template, none of the extensions should be referenced, so this using statement shouldn't exist

Reply:
You are correct. 
I miss the big picture.
![image](https://user-images.githubusercontent.com/116665025/230395194-6045e963-efa1-486e-a0d6-5b764ba33bd5.png)